src/ape/api/providers.py

```diff
@@ -70,17 +71,31 @@ class BlockConsensusAPI(BaseInterfaceModel):
 
 class BlockAPI(BaseInterfaceModel):
     """
-    An abstract class representing a block and its attributes.
+    An abstract class representing block consensus-data,
```

Comment:
Is this a copy-paste error? The `BlockAPI` contains more than just `consensus-data`.